### DISCUSSION TITLE
Let us know when you are ready for GUI and layout comments.

### DISCUSSION THREAD

**Discussion by EmilioRoberto** [soundshed/soundshed-app#8] [Ideas]
As I stated I can't offer any help with the programing/technical side of things, but if you are open to ideas about the layout and such, when you are ready I have some ideas.

**Comment by webprofusion-chrisc** [comment]
Hi Emilio, yes very keen to hear ideas and comments. I'm not particularly set on how it should look or what it should do, it just depends how much work I have to do!
